db/migrate/20181129173239_create_books.rb

```diff
@@ -0,0 +1,13 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :genre
```

Comment:
Add, null constraints. It's good to have them despite the rails validations

Reply:
done